File: app/src/main/java/oleg/osipenko/mai/data/api/SingleNews.java
package oleg.osipenko.mai.data.api;

/**
 * Created by olegosipenko on 03.09.15.
 */
public class SingleNews {
    private String header;
    private String photo;
    private String date;
    private String author;
    private String body;

    public String getHeader() {
        return header;
    }

    public String getPhoto() {
        return photo;
    }

    public String getDate() {
        return date;
    }

    public String getAuthor() {
        return author;
    }

    public String getBody() {
        return body;
    }

    @Override
    public String toString() {
        return "SingleNews{" +
                "header='" + header + '\'' +
                ", photo='" + photo + '\'' +
                ", date='" + date + '\'' +
                ", author='" + author + '\'' +
                ", body='" + body + '\'' +
                '}';
    }
}

File: app/src/main/java/ru/mai/app/data/api/SingleNews.kt
package ru.mai.app.data.api

/**
 * Created by olegosipenko on 03.09.15.
 */
class SingleNews {
    val header: String? = null
    val photo: String? = null
    val date: String? = null
    val author: String? = null
    val body: String? = null

    override fun toString(): String {
        return "SingleNews{header='$header', photo='$photo', date='$date', author='$author', body='$body'}"
    }
}
